pset-functions.js

```diff
@@ -208,6 +406,31 @@ console.log('Problem 5 - bonus:')
 
 // Add your code below this line
 
+function wordCount(phrase) {
+  const wordsInPhrase = phrase.split(' ')
+  const tallyOfWords = {}
+
+  wordsInPhrase.forEach((word) => {
```

Comment:
This is a brilliant combination! A forEach method that uses the current element as the object key in bracket notation; really well done!

Justin, really great job completing every problem in this set! Keep up the good work!